Service/InstallationService.php

```diff
@@ -63,6 +66,8 @@ public function composerupdate(): int
             $this->install($plugin['name']);
         }
 
+        $this->cacheService->warmup();
```

Comment:
Lets not do this. we already have this on version-kiss. merge this to master instead!
if ($this->io) {
    $this->cacheService->setStyle($this->io);
    $this->cacheService->warmup();
}